.github/actions/spelling/expect.txt

```diff
@@ -347,6 +347,7 @@ eabcd
 eabihf
 eay
 ebfcee
+ecdc
```

Comment:
What needs this added?

Reply:
A code comment in a shell script for build msi package:

```
# "0.29.0.custom.a28ecdc" -> "0.29.0"
```

, it flagged "ecdc"

I liked having a specific example of a shorthash as opposed to "<version>.custom.<git shorthash>" and was resisting changing my comment because of a false flag of a spell checker.

Reply:
I can double check. The only suggestions I saw in the error output it gave, were to ignore whole files, or add exceptions for words.

Reply:
I believe you could add a pattern to `patterns.txt`?

Reply:
attempted to address in https://github.com/vectordotdev/vector/pull/16281/commits/e354b138153f40e17347079ec12ad321acad4602